packages/web/src/public-site/pages/landing-page/components/FeaturedContent.tsx

```diff
@@ -230,40 +183,25 @@ const FeaturedContent = (props: FeaturedContentProps) => {
           <h4 className={styles.subTitle}>{messages.subTitle}</h4>
         </animated.div>
         <div className={styles.tracksContainer}>
-          {trendingPlaylistsResponse.value == null ||
-          trendingPlaylistsResponse.value.length < 4
-            ? FALLBACK_PLAYLISTS.map((p) => (
```

Comment:
do we not need the fallback playlists anymore?

Reply:
Naw I think we good